classic/create.go

```diff
@@ -234,15 +234,34 @@ func customizeClassicChroot() error {
 		return fmt.Errorf("failed to cleanup classic /run dir: %s (%s)", err, output)
 	}
 
+	// Add SUDO_USER to sudo group in the chroot to ensure that
+	// sudo work for the calling user.
+	sudoUser := os.Getenv("SUDO_USER")
+	if sudoUser != "" {
+		// We need to use "runInClassicEnv" so that we get the
+		// bind mount of the /var/lib/extrausers directory.
+		// Without that the "SUDO_USER" will not exist in the chroot
+		if err := runInClassicEnv("usermod", "-a", "-G", "sudo", sudoUser); err != nil {
+			return fmt.Errorf("failed to add %s to the sudo users: %s", sudoUser, err)
+		}
+	}
+
```

Comment:
Not sure about what's the vanilla state of an installed system these days, but FWIW my main local user has local access because I'm in the admin group, rather than sudo. Both are in sudoers by default, IIRC. Again, not a very reliable data point, but makes me wonder about what the plain image in 16.04 is likely to have.